library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -93,6 +94,23 @@ protected function assembleTitle(BaseHtmlElement $title): void
             $this->createSubject(),
             Html::tag('span', ['class' => 'state-text'], $this->state->getStateTextTranslated())
         ));
+
+        if ($this->state->affects_children) {
+            $total = $this->item->affected_children;
+
+            if ((int) $total > 1000) {
+                $total = '1000+';
+            }
+
+            $icon = new Icon(Icons::UNREACHABLE);
+
+            $title->addHtml((new StateBadge([$icon, Text::create($total)], ''))
+                ->addAttributes([
+                'class' => 'affected-objects',
+                'title' => sprintf('%s %s', $total, t('affected objects'))
```

Comment:
fix the indentation

Reply:
and: `sprintf(t('%s affected objects'), $total)`
but: `sprintf(t('Up to %s affected objects'), $total)` is probably really better. I'd like to prevent issues (*there are just 999 objects affected!!11*) already in the first place...